setup.py

```diff
@@ -0,0 +1,16 @@
+from setuptools import setup
+
+setup(
+    name='Minetests',
+    version='0.0.1',
+    description='Complex environments based on Minetest.',
+    author='EleutherAI',
+    author_email='',
+    packages=['minetests'],
+    install_requires=[
```

Comment:
Do we need a specific protobuf version in the requirements?

Reply:
I added the pypi protobuf version I am using currently